frontend/src/components/news-page/News.tsx

```diff
@@ -1,35 +1,46 @@
 import React, { useState, useEffect } from "react";
 import { getTopWhaleVideos, RedditVideo } from "../../clients/redditApiClient";
+import { NewsCard } from "./NewsCard";
+import "./News.scss";
 
 export const News: React.FunctionComponent = () => {
   const [articlesList, setArticlesList] = useState<RedditVideo[]>();
+  const [pageStatus, setPageStatus] = useState<boolean>(true);
 
   useEffect(() => {
     const getVideos = async () => {
-      const videos = await getTopWhaleVideos();
-      setArticlesList(videos);
+      try {
+        const videos = await getTopWhaleVideos();
+        setArticlesList(videos);
+      } catch (err) {
+        console.log("error", err);
```

Comment:
Please use `console.error()` for errors, and be more descriptive! Maybe "Error encountered when loading list of videos"?